package-build/debian-libcpr/copyright

```diff
@@ -14,6 +14,7 @@ Files:     .clang-format
            cpr-config.cmake
            cpr/*
            include/*
+           nuget/*
```

Comment:
Why should we include the nuget stuff inside the deb package?

Reply:
A Debian source package has at the heart the original upstream tarball.  If these files would be present in released tarballs, it's best that the copyright file reflect that.

As to the binary packages produced by the build process, the nuget directory is entirely irrelevant :)

If this entry isn't added, the [lintian](https://wiki.debian.org/Lintian) lint tool will complain.

In summary this is just about tidiness and has no outcome on the binary packages.